frontend/views/containers/chatroom/MessageBase.vue

```diff
@@ -235,6 +237,12 @@ export default ({
       } else if (!this.isEditing) {
         this.openMenu()
       }
+    },
+    getTimeStamp (datetime) {
+      const tString = humanTimeString(datetime, { hour: '2-digit', minute: '2-digit', hour12: true })
+      // Extract only numbers and colons, removing non-numeric characters like AM/PM
+      const cleaned = tString.replace(/[^\d:]/g, '')
+      return cleaned
```

Comment:
With 12-hour format, `date. toLocaleTimeString` return string that says whether AM or PM. eg)
```
console.log(date.toLocaleTimeString("en-US"));
// "7:00:00 PM"

console.log(date.toLocaleTimeString("ko-KR"));
// "오후 12:00:00"
```
reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString#using_locales

This info does not fit the available space and is better to be removed.